apps/omg_db/test/omg_db/release_tasks/set_key_value_db_test.exs

```diff
@@ -40,11 +40,11 @@ defmodule OMG.DB.ReleaseTasks.SetKeyValueDBTest do
     :ok = System.put_env("DB_PATH", "/tmp/YOLO/")
     :ok = SetKeyValueDB.init([])
     configuration = Enum.sort(Application.get_all_env(@app))
-    "/tmp/YOLO/watcher" = configuration[:path]
+    "/tmp/YOLO/watcher_info" = configuration[:path]
```

Comment:
Why the change? Just in case if there was an issue I should beware of.

Reply:
because of the DB path: https://github.com/omisego/elixir-omg/pull/1027/files#diff-2fb51e4c80ccbd9db57164cb0023d00aR44

if you run things barebone locally, watcher and watcher_inf would point to the same directory